src/main/resources/application.yml

```
@@ -2,29 +2,63 @@ spring:
-    url: jdbc:h2:mem:testdb
-    driver-class-name: org.h2.Driver
-    username: sa
-    password:
+    url: jdbc:mysql://localhost:3306/minipay?serverTimezone=Asia/Seoul
+    driver-class-name: com.mysql.cj.jdbc.Driver
+    username: sihyun
+    password: plate0113!
+    hikari:
+      maximum-pool-size: 20     # 동시에 사용 가능한 최대 DB 연결 수
+      minimum-idle: 5           # 최소 유휴 커넥션 수 (항상 대기 상태로 유지)
+      connection-timeout: 30000 # 커넥션 풀에서 커넥션을 얻기 위한 최대 대기 시간 (30초)
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)